api/src/com/cloud/agent/api/storage/OVFHelper.java

```diff
@@ -113,7 +113,7 @@ public static Long getDiskVirtualSize(Long capacity, String allocationUnits, Str
                 String allocationUnits = disk.getAttribute("ovf:capacityAllocationUnits");
                 od._diskId = disk.getAttribute("ovf:diskId");
                 od._fileRef = disk.getAttribute("ovf:fileRef");
-                od._populatedSize = Long.parseLong(disk.getAttribute("ovf:populatedSize") == null ? "0" : disk.getAttribute("ovf:populatedSize"));
+                od._populatedSize = NumberUtils.toLong(disk.getAttribute("ovf:populatedSize"));
```

Comment:
@nvazquez this can cause exception still -- this should be used with a default value like `0L` in case of error. For example like this -- `NumberUtils.toLong("string-param", 0L) `. Also no tests were ran /cc @DaanHoogland

Reply:
@rhtyd NumberUtils.toLong(Sting) calls NumberUtils.toLong(Sting,0L)

Reply:
sorry @rhtyd , you are right about the tests. I saw @borisstoyanov 's lgtm and assumed.

Reply:
@DaanHoogland cool, I did not know that internally it falls to ` NumberUtils.toLong(String,0L)`

Reply:
remains teh question: are we settling for the ci on this one (i did check that before merging) @rhtyd @rafaelweingartner @borisstoyanov ?